crates/matrix-sdk-base/src/store/mod.rs

```diff
@@ -340,6 +340,9 @@ pub struct StateChanges {
     /// A map from room id to a map of a display name and a set of user ids that
     /// share that display name in the given room.
     pub ambiguity_maps: BTreeMap<OwnedRoomId, BTreeMap<String, BTreeSet<OwnedUserId>>>,
+
+    /// A map of `RoomId` to pinned `OwnedEventId`s for each room.
+    pub pinned_events: BTreeMap<OwnedRoomId, Vec<OwnedEventId>>,
```

Comment:
I don't think this should be here: this is not stored, and this struct is meant to include things that are to be stored. Instead, you probably want a local variable in the one function that makes use of this `pinned_events ` field.

Reply:
Removed this in https://github.com/matrix-org/matrix-rust-sdk/pull/3747/commits/eafed32f64566eedd2c0edb0fe200fd3b7c1f672